src/components/moderation/moderation-common.ts

```diff
@@ -57,11 +57,11 @@ import { CommandSetBuilder } from "../../command-abstractions/command-set-builde
 
 export const duration_regex = /perm\b|(\d+)\s*([a-zA-Z]+)/;
 
-export const moderation_on_team_member_message: string = "Can't apply this moderation on team members";
-export const joke_responses = [
+export const joke_responses_other = ["You have no power over this user :(", "lol, nice try!", "One day, maybe ;)"];
```

Comment:
Was it intentional to include this as part of this PR?

Reply:
yeah, I just thought might as well while I'm already at it